client/src/i18n/en-US.json

```diff
@@ -892,11 +892,15 @@
       "heading": "Submission Content",
       "upload": {
         "label": "Upload",
-        "caption": "Select a supported file to be imported."
+        "caption": "Select a supported file to be imported.",
+        "file_caption": "Accepted file types: ",
+        "file_types": ".csv, .docbook, .docx, .epub, .html, .latex, .md, .odt, .rtf"
       },
       "upload_update": {
         "label": "Upload",
-        "caption": "Select a supported file to replace the existing text content."
+        "caption": "Select a supported file to replace the existing text content.",
+        "file_caption": "Accepted file types: ",
+        "file_types": ".csv, .docbook, .docx, .epub, .html, .latex, .md, .odt, .rtf"
```

Comment:
I'm not a fan of this duplication. I can't imagine these `file_caption` and `file_types` values ever needing to be different between uploading or updating an upload like I can for the other values. 

If you want to be thorough, I think this means refactoring `upload` and `upload_update`. If you want a quick way to do this, then I recommend rewriting this so that `file_caption` and `file_types` are combined into one new node, like at `content.support` for example.